autopts/wid/csip.py

```diff
@@ -181,7 +181,6 @@ def hdl_wid_20100(params: WIDParams):
 
     if 'CSIP/CL/SP/BV-03-C' in params.test_case_name or\
             'CSIP/CL/SP/BV-04-C' in params.test_case_name or\
-            'CSIP/CL/SP/BV-07-C' in params.test_case_name or\
```

Comment:
Should we add a comment stating that `CSIP/CL/SP/BV-07-C` is the only test case that will receive the WID 20101, and thus does not need to perform discovery here?